control/grpc.py

```diff
@@ -46,13 +47,13 @@ def create_bdev(self, request, context=None):
         """Creates a bdev from an RBD image."""
 
         self.logger.info(
-            f"Received request to create bdev {request.bdev_name} from"
-            f" {request.ceph_pool_name}/{request.rbd_name}"
-            f" with block size {request.block_size}")
+            f"Received request to create bdev from {request.ceph_pool_name}"
+            f"/{request.rbd_name} with block size {request.block_size}")
```

Comment:
Nit: since RPC protocol is changed here anyway, I'd be good to rename `request.ceph_pool_name` and `request.rbd_name` to `request.rbd_pool_name` and `request.rbd_image_name` respectively (with an eye towards `request.rbd_namespace_name` support in the future).